File: Ghost/Views/BlogSelectorRowView.m
//
//  BlogSelectorRowView.m
//  Ghost
//
//  Created by Enric Enrich on 15/09/15.
//
//

#import "BlogSelectorRowView.h"

@implementation BlogSelectorRowView

- (void)drawSelectionInRect:(NSRect)dirtyRect
{
    if (self.selectionHighlightStyle != NSTableViewSelectionHighlightStyleNone)
    {
        [[NSColor clearColor] setStroke];
        [[NSColor clearColor] setFill];
        
        NSBezierPath *selectionPath = [NSBezierPath bezierPathWithRect:dirtyRect];
        [selectionPath fill];
        [selectionPath stroke];
    }
}

@end

File: Ghost/Views/BlogSelectorRowView.swift
//
//  BlogSelectorRowView.swift
//  Ghost
//
//  Created by Enric Enrich on 19/09/15.
//  Copyright © 2015 Ghost. All rights reserved.
//

import Cocoa

class BlogSelectorRowView: NSTableRowView {

    // MARK: Override
    
    override func drawSelectionInRect(dirtyRect: NSRect) {
        NSColor.clearColor().setStroke()
        NSColor.clearColor().setFill()
        
        let selectionPath: NSBezierPath = NSBezierPath(rect: dirtyRect)
        selectionPath.fill()
        selectionPath.stroke()
    }
    
}
